### PR TITLE
Add support for quotes in commands and some exec helpers

### PR DESCRIPTION
**Pull request by johnewart**
Commands with quotes are supported; also adding some exec helpers to be quiet and show progress as well as log; we might consider consolidating these so they're consistent in behavior

### CODE REVIEW

**Comment by johnewart** [comment]
Addresses #207

File: plumbing/util.go [hunk 1]
```diff
@@ -48,6 +55,90 @@ func ExecToStdout(cmdString string, targetDir string) error {
...
+			}
+		}
+	}()
+
+	go func() {
+		for {
+			select {
+			default:
+			case <-dotchan:
+				fmt.Printf(".")
```
Comment (beholders-eye) [inline plumbing/util.go]: It may look better if we only show 5 dots at max, after 5 we should delete all dots and start printing them again, inplace.

Comment (johnewart) [reply plumbing/util.go]: I thought about that but, in particular with remote logs being sent back we don't have control over the terminal directly and so it makes the streamed logs look funny. We could increase the time between dots maybe; thoughts?

Comment (beholders-eye) [reply plumbing/util.go]: Yeah, maybe 10 seconds

**Pull request merged**